include/cosim/algorithm/simulator.hpp

```diff
@@ -119,6 +119,11 @@ class simulator : public manipulable
      */
     virtual void start_simulation() = 0;
 
+    /**
+     * Runs fmiXSetINI (4.2.4) functions to initialize variables' start values.
```

Comment:
We haven't made reference to the FMI spec anywhere else in this interface, nor in any other of our general interfaces, because they are supposed to be agnostic with respect to the underlying binary interface. All FMI-specific stuff is hidden away in the `cosim::fmi` module. (I admit that we're still leaking FMI details like a sieve, but we haven't been this explicit about it before.) I suggest writing a more general and extensive description of what the function is supposed to do instead.

Also, as commented elsewhere, the functions are generally in call order, so this should go before `setup()`.